R/mod_export_data.R

```diff
@@ -196,23 +201,44 @@ mod_export_data_server <- function(input, output, session, conn) {
       value = info$include_column_names[[input$table_list]]
     )
   })
-  
+
+  observe({
```

Comment:
We do not use the observe() function here. Please use observeEvent() to monitor a single event.